docs/internals/git-workflow.md

```diff
@@ -63,7 +71,13 @@ Some tests require additional databases to be set up and configured. You can cre
 settings that are configured in `tests/data/config.php`.
 
 You may limit the tests to a group of tests you are working on e.g. to run only tests for the validators and redis
-`phpunit --group=validators,redis`. You get the list of available groups by running `phpunit --list-groups`. 
+`phpunit --group=validators,redis`. You get the list of available groups by running `phpunit --list-groups`.
```

Comment:
Tip: you can add `?w=0` to any github ULR that shows a diff to exclude whitespace.